src/index.html

```diff
@@ -20,8 +21,413 @@
       href="styles/index.scss"
     />
   </head>
-
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo"
+      >
```

Comment:
Fix code style everywhere if you have only 2 attributes
```suggestion
      <a href="#home" class="header__logo">
```